src/server/command/worldedit/blueprint.zig

```diff
@@ -103,9 +102,9 @@ fn sendInfoAndLog(comptime fmt: []const u8, args: anytype, user: *User) void {
 }
 
 fn openBlueprintsDir(source: *User) ?Dir {
-	return openDir("blueprints") catch |err| blk: {
+	return main.files.cubyzDir().openDir("blueprints") catch |err| {
```

Comment:
You are migrating more than the saves here. I agree it would make sense to also have the blueprints in a global location, but they would need to migrated (and mentioned in the PR description) as well.

Reply:
Also please remove all the folders you migrate from the .gitignore.